DOCKER.md

```diff
@@ -138,20 +150,40 @@ Replace `${HOME}/bundles` with the actual path to your bundles directory if need
 
 ```bash
 # Using echo to send a request to initialize a bundle
-echo '{"jsonrpc":"2.0","id":"1","method":"call_tool","params":{"name":"initialize_bundle","arguments":{"source":"/data/bundles/bundle.tar.gz"}}}' | ./scripts/run.sh
+echo '{"jsonrpc":"2.0","id":"1","method":"call_tool","params":{"name":"initialize_bundle","arguments":{"source":"/data/bundles/bundle.tar.gz"}}}' | \
+docker run -i --rm \
+  -v "$(pwd)/bundles:/data/bundles" \
+  -e SBCTL_TOKEN="$SBCTL_TOKEN" \
+  mcp-server-troubleshoot:latest
```

Comment:
I like this example I've never tested this can you validate that this does return something? Maybe instead of initialize_bundle this should be list_bundles which will return even if there isn't a valid bundle to use.

Alternately use the bundle in /tests/fixtures to ensure that this command does run properly if we're going to document it for people.